commands/update.js

```diff
@@ -0,0 +1,58 @@
+const exec = require('child_process').exec;
+
+module.exports = {
+  describe: 'Update the SparkPost CLI',
+  action: function(callback) {
+    let packageName = require('../package.json').name;
+
+    isFromNPM(packageName, function(fromNPM) {
+      if (!fromNPM) {
+        return callback(new Error('Cannot update CLI via NPM.\nIf you cloned it from github pull the latest version to update.'));
+      }
+      else {
+        update(packageName, callback);
```

Comment:
This branch doesn't `return` which is presumably harmless but possibly good form to include it?